pipeline/azure-pipelines.yml

```diff
@@ -316,6 +327,14 @@ jobs:
     displayName: Publish Quality Gate Result (VS2022 only)
     condition: and(succeeded(), eq(variables['vsTargetVersion'], '2022'))
 
+  - task: JFrogBuildPromotion@1
+    inputs:
+      artifactoryConnection: 'repox_promoter_token'
+      buildName: '$(buildName)'
+      buildNumber: '$(Build.BuildId)'
+      targetRepo: 'sonarsource-public-builds'
+      status: 'it-passed'
```

Comment:
`it`?

Reply:
Integration tests passed, it's a standard status used on repox when an artifact passed the tests and getting promoted from `qa` to the `builds`.